### PR TITLE
Refactor and cleanup

### PR DESCRIPTION
**Pull request by sammcj**
- Add `Gemfile`
- Add `.gitignore` for common ruby files
- Correct ruby file extension
- Correct shebang
- Correct lookup of hostname to be FQDN / DNS name rather than the shortname

### CODE REVIEW

File: check_gluster.rb [hunk 1]
```diff
@@ -1,13 +1,4 @@
-#!/usr/bin/ruby -w
-
-# We use old ruby 1.8.7
-# rubocop:disable Style/HashSyntax
-
-# We match all exceptions except SystemExit to redefine exit status for nagios
-# rubocop:disable Lint/RescueException
```
Comment (atlantos) [inline check_gluster.rb]: Removing rubocop lines causing a lot of warnings while checking code with rubocop. Can we leave these rubocop custom configuration?

File: check_gluster.rb [hunk 2]
```diff
@@ -180,7 +171,7 @@ def volume_info(volume)
     def volume_status(volume, bricks, checks)
       nodes = gluster("volume status #{volume}")['volStatus']['volumes']['volume']['node']
       nodes = any_to_array(nodes)
-      nodes.each { |node| node['path'] = Socket.gethostname if node['path'] == 'localhost' }
+      nodes.each { |node| node['path'] = Socket.gethostbyname(Socket.gethostname).first if node['path'] == 'localhost' }
```
Comment (atlantos) [inline check_gluster.rb:180]: This seems fine for me

Comment (atlantos) [reply check_gluster.rb:180]: @sammcj, could you please leave rubocop rules? Then I will merge this PR.

**Review by atlantos — CHANGES_REQUESTED** [review]

File: check_gluster.rb (hunk 2 above)
Comment (sammcj) [reply check_gluster.rb:180]: Rubocop linting disablers returned

**Pull request merged**